Cargo.toml

```diff
@@ -41,10 +41,10 @@ default-features = false
 # breaking semver bumps very frequently.
 #
 # Also, sprs has an ndarray dependency that might not be the most recent.
-ndarray = { version = "0.14" }
-sprs = { version = "0.10", default-features = false }
+ndarray = { version = "0.15" }
+sprs = { version = "0.11", default-features = false }
 bencher = { version = "0.1" }
-zip = { version = "0.5" }  # NOTICE: also in dependencies
+zip = { version = "0.6" }  # NOTICE: also in dependencies
 
 [features]
 default = []
```

Comment:
No appropriate line to put this on the diff so I'll just put this here to remind myself:  The documention of the `derive` feature in `lib.rs` should be changed to no longer say that it adds build-time dependencies on `syn`/`quote`/etc.  (you can fix this or I'll fix it after)